rpm/python-oq-engine.spec.inc

```diff
@@ -43,6 +46,12 @@ available the OpenQuake Engine default celeryconfig.py, located in
 
 Copyright (C) 2010-2016, GEM Foundation.
 
+%pre
+getent group %{oquser} >/dev/null || groupadd -r %{oquser}
+getent passwd %{oquser} >/dev/null || \
+    useradd -r -g %{oquser} -m -d /var/lib/%{oquser} -s /sbin/nologin \
+    -c "The OpenQuake user" %{oquser}
+exit 0
```

Comment:
It is best for openquake to be a normal user (i.e. we can log in as openquake to debug the web ui or the dbserver).